sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2916,6 +2916,28 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("SPARK-36086: CollapseProject project replace alias should use origin column name") {
```

Comment:
@cloud-fan Add new UT in CollapseProjectSuite and should we remove this UT?

Reply:
yea we can remove

Reply:
> yea we can remove

Done